site/content/docs/5.3/utilities/borders.md

```diff
@@ -122,6 +133,12 @@ Or, choose from any of the `.border-opacity` utilities:
 
 ## Width
 
+{{< design-callout-alert >}}
+These border variants with **a width of 0px, 1px and 3px** should not be used because they do not respect the Orange Design System specifications.
```

Comment:
```suggestion
The border variant with **a width of 3px** should not be used because they do not respect the Orange Design System specifications.
```
I'd rather make it this way